bot/commands/cmd_wiki.go

```diff
@@ -52,6 +53,38 @@ func (cm *CommandManager) commandWiki(
 		_, err = cm.Discord.ChannelMessageSend(
 			message.ChannelID,
 			"SA:MP Wiki | "+args+"\n"+wikiURL)
+
+		doc, err := goquery.NewDocument(wikiURL)
+		if err == nil {
```

Comment:
Why is this `err == nil` and not `err != nil` with an early-return guard clause?

Reply:
Would it make any difference? I'm not used with go. Early return seems to more clear and neat. Is there any other advantage?

Reply:
Changed them to early return.Thanks :)